### ISSUE TITLE
Add Files view within the app

### ISSUE THREAD

**Issue by goerz** [holzschu/Carnets#51]
Would it be possible to add a Files view *within* the app?

Currently (as far as I can tell), it really only makes sense to open Carnets from the Files app, through the Share menu on particular notebook files (or just "open", but which app will open a given file seems rather unpredictable).

I would expect to be able to *close* notebooks that are open in Carnets and to get to a Files browser - pretty much like Juno's interface, in that particular respect.

This would also give Carnets a place for some additional UI elements in the future, like Settings / package management.

**Comment by holzschu** [comment]
I'm not sure I understand this one. The app begins with a Files view, and whenever you have a notebook open, if you click the back button (top left), you go back to this Files view with the ability to browse files.

**Comment by goerz** [comment]
Huh, that's weird. Something must have gotten messed up in my installation. I uninstalled and then reinstalled Carnets, and now it behaves like you say (which is exactly how I think it *should* behave).

Before, it it didn't show any Files view (or "Documents browser" is the right term, I think) , and there was no back button. Starting Carnets directly as an app would only show the last notebook, or a blank page. The only way to open files in Carnets was to go through the actual Files app. Maybe some nbextension confused the interface? I'll be keeping an eye on that, but for now, I think we can close this issue. Reinstalling also solved #50. Moreover, the fact that before I had to go through the Files app, instead of having a Document browser inside Carnets was probably the root of my problems with authorizing an external folder. I'll test that again.
